dll/cpl/input/settings_page.c

```diff
@@ -10,195 +10,358 @@
 #include "layout_list.h"
 #include "locale_list.h"
 #include "input_list.h"
+#include <shellapi.h>
```

Comment:
For which function(s) is this header needed?

Reply:
`ExtractIconExW`.

Reply:
Isn't it possible to use LoadImageW or LoadIconW instead? Those two functions are in user32 and this would allow not getting an otherwise useless dependency on the shell. It looks to me those functions also allow a path to a file to be used.

Reply:
Correct, that Isn't possible.
`ExtractIconExW` is doing a lot more than just calling `LoadIconW`

Reply:
Fixed. Unlinked `shell32`.